LanguageFeatures/Class-modifiers/basic_restrictions_A04_t27.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion It is a compile-time error if:
+/// ...
+/// - A declaration has a base or final superdeclaration, and is not itself
+///   marked base, final or sealed.
+///
+/// @description Check that it is a compile-time error if a declaration is not
+/// `base`, `final` or `sealed` and has a superdeclaration marked `sealed`. Test
```

Comment:
If I understood the specification correctly, the restriction imposed by the `sealed` declarations is that they can be extended only in the same library they are declared. It is a compile-time error to use them as superdeclarations outside of their own library, regardless of the modifiers `base`, `final`, and `sealed`.

Reply:
Indeed. And we have `basic_restrictions_A01_t01.dart` (`basic_restrictions_A01_t09.dart` for type aliases) testsing that  it's an error to extend a `sealed` class outside of its library by a class with any modifier. Deleted this test. Thank you for noticing